packages/ButtonGroup/ButtonGroupItem/ButtonGroupItem.modules.scss

```diff
@@ -0,0 +1,40 @@
+@import '~@tds/core-colours/colours';
```

Comment:
Should we be using rem?

Reply:
Not sure which line you're referring to here.

Reply:
Everywhere we use px.

Reply:
Ahh, well here it's ok because they're used for box shadows. The box shadows here were defined as `px`, meaning that they're absolute values. Regardless of zoom level, we want them to keep their pixel count.